### PR TITLE
Add hybridcontents

### PR DESCRIPTION
**Pull request by devstein**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

CC @yzq89

### CODE REVIEW

**Comment by devstein** [comment]
@conda-forge-admin This is my first time contributing. Any help or advice is appreciated!

**Comment by npavlovikj** [comment]
So, conda supports selectors, https://conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors, defined with `[]`. 
You do not use selectors, but you do have `notebook[test]` and `hybridcontents[test]` in your recipe, and that is detected as a selector, thus the error.
I would suggest you to modify the recipe and use `notebook` and `hybridcontents` instead (those are the packages installed anyway).

**Comment by devstein** [comment]
@npavlovikj Fixed, thanks for the detailed response!

File: recipes/hybridcontents/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - notebook >=4.0
```
Comment (npavlovikj) [inline recipes/hybridcontents/meta.yaml]: Usually, having `pip` and `python` is enough in the `host` section. Can you please try removing `notebook`, `six` and `requests` from here, and leaving them in `run`?

File: recipes/hybridcontents/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  imports:
+    - hybridcontents
+  requires:
+    - pytest
+    - mock
+    - nose
+  source_files:
+    - hybridcontents/tests/*.py
+  commands:
+    - pytest hybridcontents/tests/
```
Comment (npavlovikj) [inline recipes/hybridcontents/meta.yaml]: Please see https://conda-forge.readthedocs.io/en/latest/testing.html#pytest-tests, I think you should use `pytest --pyargs hybridcontents/tests/` instead.

File: recipes/hybridcontents/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+about:
+  home: https://github.com/viaduct-ai/hybridcontents
+  license: Apache Software
```
Comment (npavlovikj) [inline recipes/hybridcontents/meta.yaml]: Use `Apache-2.0` instead.

File: recipes/hybridcontents/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "Hybrid Content Manager"
+
```
Comment (npavlovikj) [inline recipes/hybridcontents/meta.yaml]: Remove empty line.

File: recipes/hybridcontents/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  # The remaining entries in this section are optional, but recommended.
```
Comment (npavlovikj) [inline recipes/hybridcontents/meta.yaml]: Remove comment.

File: recipes/hybridcontents/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    - hybridcontents/tests/*.py
+  commands:
+    - pytest hybridcontents/tests/
+
+about:
+  home: https://github.com/viaduct-ai/hybridcontents
+  license: Apache Software
+  license_family: APACHE
+  license_file: LICENSE
+  summary: "Hybrid Content Manager"
```
Comment (npavlovikj) [inline recipes/hybridcontents/meta.yaml]: I think "A ContentsManager wrapper for using multiple ContentsManager in Jupyter" is better suited for the `summary` here.

File: recipes/hybridcontents/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+  # The remaining entries in this section are optional, but recommended.
+  description: "A ContentsManager wrapper for using multiple ContentsManager in Jupyter"
```
Comment (npavlovikj) [inline recipes/hybridcontents/meta.yaml]: You can leave this description, or use something else from the homepage, e.g.: "The HybridContentManager was originally created by Quantopian as part of pgcontents; however, the usage of HybridContentsManager was restricted to the compatibility requirements of pgcontents. These restrictions included postgres dependencies and no support for the latest notebook version (>6). At Viaduct we used pgcontents exclusively for the HybridContentsManager and wanted to extend its functionality, so we created this fork hybridcontents." etc.

**Comment by npavlovikj** [comment]
@devstein , I left few minor comments. Please address those, and we can merge the recipe after.

File: recipes/hybridcontents/meta.yaml (hunk 7 above)
Comment (devstein) [reply recipes/hybridcontents/meta.yaml]: Updated

File: recipes/hybridcontents/meta.yaml (hunk 1 above)
Comment (devstein) [reply recipes/hybridcontents/meta.yaml]: Worked!

**Comment by devstein** [comment]
@npavlovikj Addressed your comments

**Comment by npavlovikj** [comment]
Thank you for the prompt fixes, and contributing @devstein 🎉

**Pull request merged**